catalog/ui/search-ui/standard/src/main/webapp/templates/menu/menubarLoginExternal.handlebars

```diff
@@ -0,0 +1,16 @@
+{{!--
```

Comment:
Should this file just be removed now that we are showing "anonymous" and a "log out" button instead?